index.html

```diff
@@ -4380,6 +4381,125 @@ <h3>Parsing content</h3>
       [[RFC2048]] and [[RFC2046]].
     </p>
   </section>
+  <section> <h3>Privacy implications and implementation considerations</h3>
+    <p>
+      The comparison to barcodes or QR codes is appropriate because NFC tags
+      are another non-human-readable method of exchanging data and sharing
+      them can have unforeseen privacy and security implications. For a web
+      site to read a QR code, a piece of interruptive UI must be used (the
+      camera) which captures an image and makes it apparent that the contents
+      of this image (including the QR code) will be available to the web page,
+      making it clear to the user that scanning is being performed.
+    </p>
+    <p>
+      Scanning a tag with NFC requires the user to place the scanning device
+      (e.g. phone) in close proximity to the NFC tag.
+    </p>
+    <p>
+      Scanning a tag when a Web NFC scan is not active, triggers the host OS
+      handling. Thus launching URLs and apps from scanning a NFC tag is not
+      handled and supported by Web NFC itself.
+    </p>
+    <p>
+      Furthermore, Web NFC scanning needs to be activated from a user interaction,
+      and scanning is paused when the web site is not in focus or the device
+      screen turns off (i.e. is not unlocked). This is put in place so that
+      accidental scans do not happen.
+    </p>
+    <p>
+      Web NFC further recommends that the implementation makes it very clear
+      UX-wise to the user that data will be scanned when placing the scanning
+      device in close proximity to a NFC tag - basically mimicking the UX flow
+      of scanning a QR code.
+    </p>
+    <p>
+      There are many options for doing so, like playing back a sound or showing
+      some persistent UI while the scanning can happen, for instance a modal
+      dialog with the ability to cancel at any point.
+    </p>
+    <img src="images/scan.jpg">
+    <p>
+      An implementation could also show the data that is about to be uploaded,
```

Comment:
Binary data would be hard to show to users. However, it would be great to let users pick which records to share eventually.

Reply:
It might just list that the tag contains "images, text and binary data"